hathor/indexes/rocksdb_tokens_index.py

```diff
@@ -229,6 +237,7 @@ def _create_genesis_info(self) -> None:
             self._settings.HATHOR_TOKEN_UID,
             self._settings.HATHOR_TOKEN_NAME,
             self._settings.HATHOR_TOKEN_SYMBOL,
+            None,
```

Comment:
I guess we can internally have a third type `NATIVE_TOKEN` that would differ from both deposit and fee based tokens.